docs/control-structures.rst

```diff
@@ -52,40 +57,53 @@ write::
 
 The names of output parameters can be omitted.
 The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
+
+The ``return`` statements are also capable of returning multiple values using ``return (v0, v1, ..., vn)``. The number of parameters must equal the number of output parameters and they must all be the same type.
+
 Return parameters are initialized to zero; if they are not explicitly
 set, they stay to be zero.
 
 Input parameters and output parameters can be used as expressions in
 the function body.  There, they are also usable in the left-hand side
 of assignment.
 
+.. note::
+    You cannot return some types from non-internal functions. If you enable the
+    new experimental ``ABIEncoderV2`` feature by adding ``pragma experimental
+    ABIEncoderV2;`` to your source file then more types are available, but
+    ``mapping`` types are still limited to inside a single contract and you
+    cannot transfer them.
+
 .. index:: if, else, while, do/while, for, break, continue, return, switch, goto
 
 Control Structures
 ===================
 
-Most of the control structures from JavaScript are available in Solidity
-except for ``switch`` and ``goto``. So
-there is: ``if``, ``else``, ``while``, ``do``, ``for``, ``break``, ``continue``, ``return``, ``? :``, with
-the usual semantics known from C or JavaScript.
+Most of the control structures of curly-braces languages such as JavaScript or C are available in Solidity except for ``switch`` and ``goto``. The full list is:
 
-Parentheses can *not* be omitted for conditionals, but curly brances can be omitted
-around single-statement bodies.
+- ``if (<condition>) {<true-branch>}``
+  - **Alternative syntax**: ``if (<condition>) <true-statement>``
```

Comment:
Can you create a new PR with only this paragraph?

I have the feeling that the alternative syntax makes the list a bit less clean. Perhaps we can first give the list just with statements and then add a paragraph after the list that states that statements can always be replaced with blocks and provides one or two examples.